.github/workflows/code-quality.yaml

```diff
@@ -68,7 +68,7 @@ jobs:
         shell: bash
         working-directory: icechunk-python
         run: |
-          uvx --with mypy --with zarr --with hypothesis --with xarray --with dask mypy python
+          uv run --no-project mypy python tests
```

Comment:
❤️ if this works. very slick. the power of actually reading the docs :)

Reply:
It didn't I've updated the commit message with the CI error, I'm trying a second one lol